### PR TITLE
Add `debug`; add a test

### LINKED ISSUE #31: debug/debugShow is missing
`console.{log,warn,error}` are implemented with bindings.
`console.debug` is missing.

Is there a reason? Can it be added / would a PR be welcome?

https://developer.mozilla.org/de/docs/Web/API/Console/debug

### PR DESCRIPTION
**Pull request by zyla**
**Description of the change**

Add a wrapper for `console.debug`.

Also added a simple test script, because there was none.

Fixes #31 

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [x] Linked any existing issues or proposals that this pull request should close
- [x] Updated or added relevant documentation
- [x] Added a test for the contribution (if applicable)

### CODE REVIEW

File: scripts/test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+pulp build -I test
+node -e 'require("./output/Test.Main/index.js").main();' > test/actual_output.txt 2>&1
```
Comment (JordanMartinez) [inline scripts/test]: Could this be updated to use ES syntax?

Comment (zyla) [reply scripts/test]: Yes

**Review by JordanMartinez — APPROVED** [review]

**Comment by JordanMartinez** [comment]
:ping_pong: @thomashoneyman

**Review by thomashoneyman — APPROVED** [review]
